From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@msolhab__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (mohabib). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

carolinacmoravia committed:
C52874: Broken bookmark: [Naming and referencing containers and blobs]

@michaelhauss,
Could you review the reported bookmark? It is not working.
Many thanks for your time in advance.

carolinacmoravia committed:
Update storage-ios-how-to-use-blob-storage.md

@MichaelHauss,
Could you check if the porposed file change is correct? Thanks.

Commit:
edit pass: stream-analytics-stream-analytics-query-patterns

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into stream-analytics-stream-analytics-query-patterns2

Commit:
edit pass: stream-analytics-stream-analytics-query-patterns

LoriMerrick committed:
Update stream-analytics-stream-analytics-query-patterns.md

LoriMerrick committed:
Update stream-analytics-stream-analytics-query-patterns.md

v-vikum2 committed:
USER STORY 1048717 SaaS App Tutorial: Asana Tutorial Update

burkeholland committed:
Node.js Samples - Postgres - Remove Node Install

I think it's safe to assume that if people are on this page, they already have Node installed.

burkeholland committed:
Node.js Samples - Postgres - Clean up create table

burkeholland committed:
Update connect-nodejs.md

burkeholland committed:
Update connect-nodejs.md

burkeholland committed:
Node.js Sample - Postgres - Update Read Sample

Event Emitters are deprecated so moving to the promise model.

burkeholland committed:
Node.js Samples - Postgres - Update Data updates

burkeholland committed:
Update connect-nodejs.md

burkeholland committed:
Update connect-nodejs.md

burkeholland committed:
Update connect-nodejs.md

paulathurman committed:
edit pass: active-directory-saas-cezannehrsoftware-tutorial

ecfan committed:
Upload batching draft and art

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into batching

ecfan committed:
Address Acrolinx

ecfan committed:
Address Acrolinx

ecfan committed:
Address Acrolinx

ecfan committed:
Added notes

ecfan committed:
Address Acrolinx

ecfan committed:
Tweaks

ecfan committed:
Update TOC

ecfan committed:
Fix formatting

itechedit committed:
edit pass: active-directory-saas-vxmaintain-tutorial.md

Rajani-Janaki-Ram committed:
upgrade doc

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-saas-vxmaintain-tutorial

ecfan committed:
Tweaks

ecfan committed:
Art formatting tweaks

ecfan committed:
Art fixes

ecfan committed:
Clarification

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into active-directory-saas-cezannehrsoftware-tutorial

itechedit committed:
edit pass: active-directory-saas-vxmaintain-tutorial.md

LoriMerrick committed:
Additional changes

Changed the author and revision date per author's request.

ecfan committed:
Formatting fixes

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into active-directory-saas-vxmaintain-tutorial

itechedit committed:
edit pass: active-directory-saas-vxmaintain-tutorial.md

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into batching

paulathurman committed:
edit pass: active-directory-saas-cezannehrsoftware-tutorial

paulathurman committed:
edit pass: active-directory-saas-cezannehrsoftware-tutorial

TerryLanfear committed:
TOC update

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into unify-cloud

bwren committed:
Log Analytics Flow tutorial

spelluru committed:
new article for gateway availability and scalability

bwren committed:
images

bwren committed:
fixes

bwren committed:
fixes

bwren committed:
fixes

Rajani-Janaki-Ram committed:
upgrade doc

bwren committed:
updates and toc

ecfan committed:
Remove "and outputs"

ecfan committed:
Updated per PM feedback

ecfan committed:
Fix formatting

ecfan committed:
Fix formatting

v-viinde committed:
USER STORY 1054868 SaaS App Tutorial: InTime Update

v-viinde committed:
Merge branch 'master' into intimev1

v-viinde committed:
Updated

v-viinde committed:
Merge branch 'intimev1' of https://github.com/v-viinde/azure-docs-pr into intimev1

bwren committed:
fixes from review

bwren committed:
remove bad image

cherylmc committed:
Update vpn-gateway-certificates-point-to-site-makecert.md

bwren committed:
note

bwren committed:
fixes

cherylmc committed:
Update vpn-gateway-certificates-point-to-site-makecert.md

cherylmc committed:
Update vpn-gateway-certificates-point-to-site-makecert.md

cherylmc committed:
Update vpn-gateway-certificates-point-to-site-makecert.md

ShawnJackson committed:
Update stream-analytics-stream-analytics-query-patterns.md

ecfan committed:
Rename topic and art folder, update toc, add to examples & scenarios

ecfan committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into batching

ecfan committed:
Tweak link text

ecfan committed:
Resolve conflicts

cherylmc committed:
Update vpn-gateway-certificates-point-to-site-makecert.md

ShawnJackson committed:
Update stream-analytics-stream-analytics-query-patterns.md

rwike77 committed:
updated intro

ShawnJackson committed:
Update active-directory-saas-vxmaintain-tutorial.md

Commit:
update per usability feedback

ShawnJackson committed:
Merge pull request #18389 from LoriMerrick/stream-analytics-stream-analytics-query-patterns2

edit pass: stream-analytics-stream-analytics-query-patterns.md

TomShinder committed:
Update security-storage-overview.md

Fixed terminology to more standard.

ShawnJackson committed:
Merge pull request #18879 from itechedit/active-directory-saas-vxmaintain-tutorial

edit pass: active-directory-saas-vxmaintain-tutorial

Commit:
Merge pull request #16559 from carolinacmoravia/patch-283

C52874: Broken bookmark: [Naming and referencing containers and blobs]

rwike77 committed:
updated intro

TerryLanfear committed:
final fix

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into unify-cloud

TomShinder committed:
Update security-virtual-machines-overview.md

Small change in messaging.

sethmanheim committed:
Setup is a noun, set up is the verb

ShawnJackson committed:
Update active-directory-saas-cezannehrsoftware-tutorial.md

Commit:
Merge pull request #18973 from TomShinder/patch-24

Update security-storage-overview.md

TomShinder committed:
Update azure-operational-security-overview.md

updated intro.

Commit:
Merge pull request #18976 from TomShinder/patch-25

Update security-virtual-machines-overview.md

Commit:
Merge pull request #18972 from rwike77/dotnettutorial

updated intro

ShawnJackson committed:
Merge pull request #18915 from paulathurman/active-directory-saas-cezannehrsoftware-tutorial

edit pass: Active directory saas cezannehrsoftware tutorial

Commit:
Merge pull request #18980 from TomShinder/patch-26

Update azure-operational-security-overview.md

Juliako committed:
Update video-indexer-use-apis.md

cherylmc committed:
update settings values

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into s2s10

sethmanheim committed:
Removing mentions of deprecated ACS

Commit:
Updated graphics

ecfan committed:
Fix ToC

sethmanheim committed:
Making language stronger

TerryLanfear committed:
final, final fix

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into unify-cloud

Commit:
Merge pull request #18981 from Juliako/patch-10

Update video-indexer-use-apis.md

Commit:
Merge pull request #18965 from cherylmc/patch-1

Update vpn-gateway-certificates-point-to-site-makecert.md

Lisaco88 committed:
Merge pull request #18577 from burkeholland/patch-14

Node.js Samples - Postgres - Clean up create table

cherylmc committed:
update

Commit:
Merge pull request #18543 from burkeholland/patch-9

Node.js Samples - Postgres - Remove Node Install

Lisaco88 committed:
Merge pull request #18581 from burkeholland/patch-15

Node.js Sample - Postgres - Update Read Sample

bprakash committed:
Update hdinsight-component-versioning.md

3.6 is default now

SnehaGunda committed:
Updating command

Commit:
Merge pull request #18992 from bprakash/patch-8

Update hdinsight-component-versioning.md

Juliako committed:
Update media-services-sspk.md

Lisaco88 committed:
Merge pull request #18818 from burkeholland/patch-17

Node.js Samples - Postgres - Update Data updates

tamram committed:
task count updates

Commit:
Merge pull request #18993 from SnehaGunda/envupdates

Updating command

Commit:
Merge pull request #18995 from Juliako/patch-11

Update media-services-sspk.md

cherylmc committed:
update typo

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into s2s10

sethmanheim committed:
Adding a quick link update

Lisaco88 committed:
Merge pull request #18862 from Rajani-Janaki-Ram/master

upgrade doc

cherylmc committed:
update

Commit:
Updating pricing link in TOC

Commit:
Updating pricing link in TOC

Commit:
Updating pricing link in TOC

ecfan committed:
Tweaks

GitHubber17 committed:
Merge pull request #18984 from cherylmc/s2s10

S2s10

Ja-Dunn committed:
Merge pull request #18921 from TerryLanfear/unify-cloud

Unify cloud

ecfan committed:
Tweaks

sethmanheim committed:
Adding to TOC

Commit:
Updated TOC and get started

Commit:
Merge pull request #18996 from tamram/tamram-0809

task count updates

HJToland3 committed:
postgres usability study

tamram committed:
Update batch-get-task-counts.md

Commit:
Updated TOC title for Troubleshoot section

tamram committed:
Update batch-get-task-counts.md

bwren committed:
number fix

Commit:
Merge pull request #19005 from HJToland3/localworking

postgres usability study

Ja-Dunn committed:
Merge pull request #18975 from stevestein/SqlDb080201

update per usability feedback

Ja-Dunn committed:
Merge pull request #18978 from sethmanheim/toc-fix

Setup is a noun, set up is the verb

Ja-Dunn committed:
Merge pull request #18982 from sethmanheim/sas-fix

Removing mentions of deprecated ACS

cherylmc committed:
update

Commit:
Merge pull request #19006 from tamram/patch-1

Update batch-get-task-counts.md

Ja-Dunn committed:
Merge pull request #18983 from MikkelHegn/master

Updated graphics

Ja-Dunn committed:
Merge pull request #18998 from HeathL17/hl-TOC

Updating Pricing link

spelluru committed:
renamed the image folder. fixed an issue with the move data

Ja-Dunn committed:
Merge pull request #18942 from bwren/flow

Log Analytics Flow tutorial

Ja-Dunn committed:
Merge pull request #18987 from cherylmc/certs5

update

GitHubber17 committed:
Merge pull request #18940 from spelluru/adfgatewayscalability0808

new article for gateway availability and scalability

Ja-Dunn committed:
Merge pull request #18956 from v-viinde/intimev1

USER STORY 1054868 SaaS App Tutorial: InTime Update

Ja-Dunn committed:
Merge pull request #18435 from v-vikum2/asanaupdate

USER STORY 1048717 SaaS App Tutorial: Asana Tutorial Update

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

Ja-Dunn committed:
Merge pull request #18851 from ecfan/batching

Upload draft topic and art for batching in logic apps

GitHubber17 committed:
Merge pull request #19003 from jeffgilb/jeff89

Updated TOC and get started

v-alje committed:
Merge pull request #19013 from MicrosoftDocs/FromPublicRepo

From public repo
